lib/reporters/lcov-reporter.js

```diff
@@ -0,0 +1,29 @@
+
+var Reporter = require('../reporter');
+
+var lcovRecord = function(data) {
+
+  var str = "";
+  str += 'SF:' + data.fileName + '\n';
```

Comment:
What do 'SF:' and 'DA:' stand for?  I understand the data your putting here, just not the labelling.